components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabToolbarFeature.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.components.feature.customtabs
+
+import android.graphics.Color
+import mozilla.components.browser.session.Session
+import mozilla.components.browser.session.SessionManager
+import mozilla.components.browser.toolbar.BrowserToolbar
+import mozilla.components.support.base.feature.LifecycleAwareFeature
+
+/**
+ * Initializes and resets the Toolbar for a Custom Tab based on the CustomTabConfig.
+ */
+class CustomTabToolbarFeature(
```

Comment:
nit: should be CustomTab**s**.

Reply:
Why hold back? :D

Reply:
I can only do one rain dance a day to get a green build.